rpcs3/Emu/Cell/Modules/cellL10n.cpp

```diff
@@ -301,7 +301,7 @@ s32 _L10nConvertChar(s32 src_code, const void *src, s32 src_len, s32 dst_code, v
 s32 _L10nConvertCharNoResult(s32 src_code, const void *src, s32 src_len, s32 dst_code, vm::ptr<void> dst)
 {
 	s32 dstLen = 0x7FFFFFFF;
-	s32 result = _ConvertStr(src_code, src, src_len, dst_code, dst.get_ptr(), &dstLen, true);
+	_ConvertStr(src_code, src, src_len, dst_code, dst.get_ptr(), &dstLen, true);
```

Comment:
can't you use [[maybe_unused]] here and in other places?
I think it is dangerous to omit return values, since developers might think the value is void.
From experience his has led to bugs in the past (not only in RPCS3) or at least made it harder to debug nasty problems.

Reply:
Ok, but there are only few places and only if attribute works correctly.

Reply:
it doesn't need to be added everywhere.
Just where the return value might indicate some sort of failure.

Reply:
Done.